packages/react/src/components/ImageUploader/useImageUploader.ts

```diff
@@ -58,7 +58,9 @@ export const useImageUploader = ({
       return
     }
 
-    const fulfilledImageFiles = Array.from(files).map(file => {
+    const fulfilledImageFiles = (
+      file: File
+    ): Promise<string | ArrayBuffer | null> => {
```

Comment:
```suggestion
const readFileAsDataURL = async (file) => new Promise(resolve => {
   const reader = new FileReader()
   reader.onload = () => resolve(reader.result)
   reader.readAsDataURL(file)
})
```
함수의 역할이 바뀌어서 네이밍도 변경되어야할 것 같습니다!

구문도 더 간결하게 위처럼 변경하는 거 어떠신가요?

Reply:
[반영 완료 ](https://github.com/price-offer/offer-design-system/pull/238/commits/7906320b979e8c0c517559a48f53686fb008318f)